src/GitLink/Linker.cs

```diff
@@ -112,11 +114,24 @@ public static bool Link(string pdbPath, LinkOptions options = default(LinkOption
                     }
 
                     string rawUrl = provider.RawGitUrl;
-                    if (!rawUrl.Contains("%var2%") && !rawUrl.Contains("{0}"))
+                    if (rawUrl.Contains(RevisionPlaceholder) || rawUrl.Contains(FilenamePlaceholder))
```

Comment:
As long as we throw and log, then we don't necessarily need to go for case insensitive here.